### DISCUSSION TITLE
Alcohol + Amphetamines

### DISCUSSION THREAD

**Discussion by zelixir25** [TripSit/drugs#75] [Drug Interactions]
Amphetamines tend to reduce alcohol-induced drowsiness, which can result in increased alcohol consumption. This combination can also increase heart rate relative to either substance alone, creating more strain on the cardiovascular system. Co-administration of these substances can also result in an increased concentration of amphetamines in your system depending on route of administration and timing. Use caution with this combination.

Althobaiti, Y. S., & Sari, Y. (2016). Alcohol Interactions with Psychostimulants: An Overview of Animal and Human Studies. Journal of Addiction Research & Therapy, 7(3), 281. https://doi.org/10.4172/2155-6105.1000281

Hamida, S. B., Plute, E., Cosquer, B., Kelche, C., Jones, B. C., & Cassel, J.-C. (2008). Interactions between ethanol and cocaine, amphetamine, or MDMA in the rat: thermoregulatory and locomotor effects. Psychopharmacology, 197(1), 67–82. https://doi.org/10.1007/s00213-007-1007-5

Mendelson, J., Jones, R. T., Upton, R., & Jacob III, P. (1995). Methamphetamine and ethanol interactions in humans. Clinical Pharmacology & Therapeutics, 57(5), 559–568. https://doi.org/10.1016/0009-9236(95)90041-1

**Comment by utaninja** [comment]
Pushed to https://github.com/TripSit/drugs/issues/311
